bookserver/routers/assessment.py

```diff
@@ -143,3 +166,355 @@ async def get_history(request: Request, request_data: HistoryRequest):
     ]
 
     return make_json_response(detail=res)
+
+
+# Used by :ref:`compareAnswers`
+@router.get("/getaggregateresults")
+async def getaggregateresults(request: Request, div_id: str, course_name: str):
+    question = div_id
+
+    if not request.state.user:
+        return make_json_response(
+            status=status.HTTP_401_UNAUTHORIZED,
+            detail=dict(answerDict={}, misc={}, emess="You must be logged in"),
+        )
+
+    if course_name in (
+        "thinkcspy",
+        "pythonds",
+        "fopp",
+        "csawesome",
+        "apcsareview",
+        "StudentCSP",
+    ):
+        start_date = datetime.datetime.utcnow() - datetime.timedelta(days=90)
+    else:
+        course = await fetch_course(course_name)
+        start_date = course.term_start_date
+
+    result = await count_useinfo_for(question, course_name, start_date)
+
+    tdata = {}
+    tot = 0.0
+    for row in result:
+        tdata[row[0]] = row[1]
```

Comment:
Docs would be nice -- what is `row[0]` and `row[1]`? Perhap use `for (act, count) in result`?